src/App.js

```diff
@@ -38,9 +38,11 @@ const notifyUser = announcement => {
 }
 
 const PageRoute = ({ path, children }) => {
+  const ONLY_APPLICATION = true // CHANGE: when not application
```

Comment:
Can we actually move this to `Constants.js`? Ideally we would have these configs in one place.

Reply:
OKay okay of course sounds good! thank you!